plugins/modules/external_usergroup.py

```diff
@@ -98,6 +102,13 @@ def main():
 
         module.set_entity('entity', entity)
         module.auto_lookup_entities()
```

Comment:
The invisible entities will not be set at this point, because ansible does not know about them. So they don't interfere the auto lookup!? After lookup we put them in one of the supported fields for the usergroup.
Trying to understand how this is supposed to work. But it sounds like the following lines should move behind the foreman api. ;)
That said, i think this can actually work.

Reply:
Auto lookup ignores them as they aren't entities (but invisible entities), and then I set them manually and use the normal entity flow to pass them properly to the API.

Reply:
would using `set_entity` work there too?

Reply:
Instead of auto_lookup? Yeah, I guess.

Reply:
instead of `module.foreman_params['auth_source_ldap'] = auth_source`